libs/NfcCoreLib/lib/Nci/phNciNfc_NfceeMgmt.c

```diff
@@ -726,6 +730,10 @@ static NFCSTATUS phNciNfc_NfceeModeSetNtfHandler(void *pContext,
             PH_LOG_NCI_INFO_STR("NFCEE Mode Set process Success");
         }
     }
+    pCtx->IfNtf = pCtx->IfModeSetNtf;
+    pCtx->IfNtfCtx = pCtx->IfNtfModeSetCtx;
```

Comment:
Same as in the comment above.

Reply:
In case of NCI 2.0 the call back will be called upon Set mode Notification.